python/llm/setup.py

```diff
@@ -321,7 +321,8 @@ def setup_package():
                         "xpu": xpu_requires,  # default to ipex 2.1 for linux and windows
                         "xpu-2-0": xpu_20_requires,
                         "xpu-2-1": xpu_21_requires,
-                        "serving": serving_requires},
+                        "serving": serving_requires,
+                        "llama_cpp": ["bigdl-core-cpp==" + VERSION + ";platform_system=='Linux'"]},
```

Comment:
change to "cpp", which should imply "xpu"

Reply:
should the user use `pip install bigdl-llm[cpp]` or `pip install bigdl-llm[xpu, cpp]`?

Reply:
Yeah, now user can
```bash
# install gpu optimizations for llama-cpp backend 
pip install --pre bigdl-llm[cpp]
# install gpu optimizations for torch backend 
pip install --pre bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
# install gpu optimizations for torch backend & llama-cpp backend
pip install --pre bigdl-llm[xpu,cpp] -f https://developer.intel.com/ipex-whl-stable-xpu
```

Reply:
what's the difference between the 1st and 3rd commands?

Reply:
`pip install --pre bigdl-llm[cpp]` -> only install bigdl-llm & bigdl-core-cpp.
`pip install --pre bigdl-llm[xpu]` -> will install bigdl-llm & bigdl-core-xe-21 & bigdl-core-xe-esimd & torch & ipex and other dependencies.
`pip install --pre bigdl-llm[xpu,cpp]` -> will install above all.